logback-core/src/main/java/ch/qos/logback/core/boolex/GEventEvaluatorBase.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Logback: the reliable, generic, fast and flexible logging framework.
+ * Copyright (C) 1999-2015, QOS.ch. All rights reserved.
+ * <p>
+ * This program and the accompanying materials are dual-licensed under
+ * either the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation
+ * <p>
+ * or (per the licensee's choosing)
+ * <p>
+ * under the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation.
+ */
+package ch.qos.logback.core.boolex;
+
+import ch.qos.logback.core.util.FileUtil;
+import groovy.lang.*;
+import org.codehaus.groovy.control.CompilationFailedException;
+
+public class GEventEvaluatorBase<E> extends EventEvaluatorBase<E> {
```

Comment:
I'd think this and `IEvaluator` should possibly be `E extends DeferredProcessingAware` but I'll defer making the adjustment until there seems to be activity.